http_test.go

```diff
@@ -0,0 +1,21 @@
+package glutton
+
+import (
+	"net/http"
+	"testing"
+)
+
+func TestFormatRequest(t *testing.T) {
+
```

Comment:
```suggestion
```

Reply:
Im sorry I will switch to goland. Vs code is really killing me.